### DISCUSSION TITLE
Apps and extensions updated for Spotify 2.0?

### DISCUSSION THREAD

**Discussion by 3raxton** [3raxton/spicetify-custom-apps#30] [General]
Hey all, 

I am going to update this repo with a legacy folder and an updated extension / custom app folder for the old and newer versions of Spotify [as requested in previous issues](https://github.com/3raxton/spicetify-custom-apps-and-extensions/issues/26). 

I have mentioned each contributor below along with their extension / custom app in hopes to update this repo by learning which have been updated and which have not. Please respond to the comment with your custom app title and if it has been updated.

**Thanks for helping this repo stay strong!** 

---

#### **Please note:** if you do NOT want to receive emails about this discussion, please unsubscribe on the right-hand column of this discussion or select the types of updates you would like to receive.

**Comment by 3raxton** [comment]
Hey @JakeRoggenbuck! Can you let me know if your extension [Auto Skip Sad Songs](https://github.com/3raxton/spicetify-custom-apps-and-extensions/tree/main/auto-skip-sad-songs) has been updated and to run on Spotify 2.0?

Thanks!

**Reply by 3raxton** [reply]
Bumping this @JakeRoggenbuck

**Comment by 3raxton** [comment]
Hey @einzigartigerName! Can you let me know if your extensions [Spicetify Copy](https://github.com/3raxton/spicetify-custom-apps-and-extensions/tree/main/copy-playlists) and [Spicetify History Shortcut](https://github.com/3raxton/spicetify-custom-apps-and-extensions/tree/main/history-shortcut) have been updated to run on Spotify 2.0?

Thanks!

**Reply by nelsongillo** [reply]
Hey, I have not updated them yet.

**Reply by 3raxton** [reply]
Hey @einzigartigerName double checking before confirmation. Have any of your extensions been updated for 2.0?

**Comment by 3raxton** [comment]
Hey @pnthach95! Can you let me know if your extension [Copy to Clipboard](https://github.com/3raxton/spicetify-custom-apps-and-extensions/tree/main/copy-to-clipboard) has been updated to run on Spotify 2.0?

Thanks!

**Reply by pnthach95** [reply]
If you mean Spicetify 2.0 then yes. Use [this file](https://github.com/3raxton/spicetify-custom-apps-and-extensions/blob/main/copy-to-clipboard/copytoclipboard2.js) to run on Spicetify 2.0

**Reply by 3raxton** [reply]
Thanks for the info @pnthach95 !

**Comment by 3raxton** [comment]
Hey @non-bin! Can you let me know if your extension [Lyrics](https://github.com/3raxton/spicetify-custom-apps-and-extensions/tree/main/lyrics) has been updated to run on Spotify 2.0?

Thanks!

**Reply by 3raxton** [reply]
Hey @non-bin, I am confirming this before updating. Has your extension been updated for Spotify 2.0?

**Comment by 3raxton** [comment]
Hey @itsmeow! Can you let me know if your extension [Spicetify Canvas](https://github.com/3raxton/spicetify-custom-apps-and-extensions/tree/main/spicetify-canvas) has been updated to run on Spotify 2.0?

Thanks!

**Reply by itsmeow** [reply]
Yes, it's in the headline of the README!

**Reply by itsmeow** [reply]
The legacy version has its own branch.

**Reply by 3raxton** [reply]
Thanks for letting me know! I also appreciate your patch and work on spicetify and keeping it alive. Glad to have you as part of the community 🔥

**Comment by 3raxton** [comment]
Hey @theRealPadster! Can you let me know if your extension [spicetify-hide-podcasts](https://github.com/3raxton/spicetify-custom-apps-and-extensions/tree/main/spicetify-hide-podcasts) has been updated to run on Spotify 2.0?

Thanks!

**Reply by theRealPadster** [reply]
Yep, it's for spicetify 2.x now. The legacy version has a release, but is no longer maintained. https://github.com/theRealPadster/spicetify-hide-podcasts/releases

**Comment by 3raxton** [comment]
Hey @elijaholmos! Can you let me know if your extension [ViewPlaylistsWithSong](https://github.com/3raxton/spicetify-custom-apps-and-extensions/tree/main/view-playlists-with-song) has been updated to run on Spotify 2.0?

Thanks!

**Reply by elijaholmos** [reply]
Hey there! No, I have not ported it over, and currently have no intention to. Still rocking spotify 1.1.56 🤙

**Reply by 3raxton** [reply]
Hey @elijaholmos, I am confirming this before updating. Has your extension been updated for Spotify 2.0 since the initial discussion?

**Reply by elijaholmos** [reply]
@3raxton It still has not been updated. Thank you for checking in again

**Comment by 3raxton** [comment]
Hey @notPlancha! Can you let me know if your extension [Volume Profiler](https://github.com/3raxton/spicetify-custom-apps-and-extensions/tree/main/volume-profiles) has been updated to run on Spotify 2.0?

Thanks!

**Reply by 3raxton** [reply]
This has recently been updated

**Comment by 3raxton** [comment]
Thank you all for the details on which extensions and custom apps are legacy and which are not. Appreciate having you all as part of this repo 🔥
